pkg/resources/quota/quota.go

```diff
@@ -85,30 +85,66 @@ type quotaConfigReceiver struct {
 	Resources map[string]ResourceConfig     `json:"resources,omitempty"`
 }
 
-func GetQuota(QuotaId string, QuotaConfig *corev1.ConfigMap, retQuota *Quota, isUpdated bool) error {
+func GetQuota(quotaParam string, QuotaConfig *corev1.ConfigMap, retQuota *Quota) (string, error) {
 	allQuotas := &[]quotaConfigReceiver{}
 	err := json.Unmarshal([]byte(QuotaConfig.Data[ConfigMapData]), allQuotas)
 	if err != nil {
-		return err
+		return "", err
 	}
 	quotaReceiver := quotaConfigReceiver{}
 
+
+	//if 10 then 1M
+	//if 50 then 5M
+	//if 100 then 10M
+	//if 20 || 200 then 20M
+	//if 500 then 50M
+	//if 1 then 1k
+	// declare the string mappings
+	quotaMappings := map[string][]string{
```

Comment:
@philbrookes i'm interested in your thoughts on this approach. We use the name of the configmap entry in a few places so this mapping should handle text in the metrics and in the dashboards. and it also gives us the option to handle the 20 env var for 20M but also trial/eval options.

Reply:
If the value is not in this mapping it's not in the config map and wouldn't be handled anyway.

Reply:
I suppose we just need to make it clear how the value is mapped from the values in the drop-down through this slice and then into the names in that configmap, that could be hard to work out for someone new to the codebase.

Reply:
Yes agreed, that was my reason actually for making the names of the entries in the config map more specific, e.g 1M, 5M.... 100K and then mapping those to any value we'd expect to have in the param from ocm or in the env var to the equivelant in the config map. Does teh comment help to explain it or should I expand on that comment a bit more. Are you happy with the approach generally @philbrookes ?

Reply:
I'm not seeing why we can't add the friendly name and the dropdown value directly to the object this maps to? Happy to discuss on a call if I'm being clear.

Reply:
that's update now @philbrookes makes sense. 👍

Reply:
cool, that looks better to me now. Thanks Laura!